src/dlgIRC.cpp

```diff
@@ -106,7 +106,7 @@ dlgIRC::~dlgIRC()
         connection->close();
     }
 
-    if (mpHost->mpDlgIRC) {
+    if (mpHost && mpHost->mpDlgIRC) {
```

Comment:
Since the second variable here is (since #4308) a `QPointer<dlgIRC>` the next line that nulls it (and thus all of new lines 109 to 111) is/are *probably* redundant (as it gets cleared anywhere either before or after this destructor is executed IIRC though I can't recall which applies!)

Reply:
Sounds like it can't hurt to have this.